src/renderer/helpers/api/local.js

```diff
@@ -437,3 +449,27 @@ export function mapLocalFormat(format) {
     url: format.url
   }
 }
+
+/**
+ * @param {import('youtubei.js/dist/src/parser/classes/comments/Comment').default} comment
+ */
+export function parseLocalComment(comment, hasOwnerReplied = false, replyToken = null) {
+  return {
```

Comment:
I like this. We should probably return new objects for other api calls as well in the future (i think this would be very useful for our invidious functions)

Reply:
Yes, it will also allow us to return the data in the same format, that way we don't need separate handling for Invidious and local API data in ft-list-channel and similar components.